### ISSUE TITLE
Add all attributes, even if unavailable from API

### ISSUE THREAD

**Issue by jonschwenk** [fitnr/buoyant#6]
Hi,

Just thought you might want to know that running the first example in the README throws an error:

```
from buoyant import Buoy
buoy = Buoy('0Y2W3')
buoy.air_pressure_at_sea_level
```

```
Traceback (most recent call last):

  File "C:\Users\Jon\Anaconda3\envs\icomdata\lib\site-packages\buoyant\buoy.py", line 132, in fetch
    result = next(reader)

  File "C:\Users\Jon\Anaconda3\envs\icomdata\lib\csv.py", line 111, in __next__
    row = next(self.reader)

StopIteration


During handling of the above exception, another exception occurred:

Traceback (most recent call last):

  File "<ipython-input-14-2d366206b95b>", line 3, in <module>
    buoy.air_pressure_at_sea_level

  File "C:\Users\Jon\Anaconda3\envs\icomdata\lib\site-packages\buoyant\buoy.py", line 156, in air_pressure_at_sea_level
    return self._get('air_pressure_at_sea_level')

  File "C:\Users\Jon\Anaconda3\envs\icomdata\lib\site-packages\buoyant\buoy.py", line 114, in _get
    return self.__dict__.setdefault(observation, self.fetch(observation, as_group))

  File "C:\Users\Jon\Anaconda3\envs\icomdata\lib\site-packages\buoyant\buoy.py", line 138, in fetch
    raise AttributeError(observation)

AttributeError: air_pressure_at_sea_level
```

**Comment by fitnr** [comment]
This worked for me.
```
>>> buoy = Buoy('0Y2W3')
>>> buoy.air_pressure_at_sea_level
Observation(1014.1, 'hPa')
```

Most likely, the NOAA API was interrupted, which means that the object didn't know to create an `air_pressure_at_sea_level` attribute. This is a flaw in how the object are instantiated that I can fix up.

**Comment by fitnr** [comment]
Fixed in v0.7.0
